app/views/categories/show.html.erb

```diff
@@ -0,0 +1,19 @@
+<% provide(:title, @category.name) %>
+<div class="row">
+  <aside class="col-md-4">
+    <section class="user_info">
+      <h1>
+        <%= @category.name %>
+      </h1>
+    </section>
+  </aside>
+  <div class="col-md-8">
+    <% if @category.questions.any? %>
+      <h3>Questions (<%= @category.questions.count %>)</h3>
+      <ol class="microposts">
```

Comment:
should this be `category` instead?

Reply:
this is to show all the questions within the category, will change the class